package.json

```diff
@@ -38,6 +38,7 @@
     "moment": "^2.15.2",
     "morgan": "~1.7.0",
     "nodemailer": "~2.6.4",
+    "pg": "^5.1.0",
```

Comment:
This is the wrong way of doing it. One must NOT include `pg` when `pg-promise` is used already. See: [How to access the instance of node-postgres that's used?](https://github.com/vitaly-t/pg-promise/wiki/FAQ#how-to-access-the-instance-of-node-postgres-thats-used).